packages/core/src/services/scales-cartesian.ts

```diff
@@ -270,7 +269,10 @@ export class CartesianScales extends Service {
 		}
 
 		if (axisOptions.scaleType === ScaleTypes.TIME) {
-			const spaceToAddToEdges = Tools.getProperty(options, "timeScale", "addSpaceOnEdges");
+			const spaceToAddToEdges = Tools.getProperty(options, "axes", axisPosition, "timeScale", "addSpaceOnEdges") !== null
+				? Tools.getProperty(options, "axes", axisPosition, "timeScale", "addSpaceOnEdges")
+				: Tools.getProperty(options, "timeScale", "addSpaceOnEdges");
+
```

Comment:
If we're going to support per-axis timescale options then it makes sense to get rid of the global timeScale field, although we should probably deprecate it and warn the user.

Reply:
OK, working on it

Reply:
@ilariaventurini let me know if you think that we should keep both

Reply:
Hi, no we think that only the global `timeScale` is necessary. In line 273 of `scales-cartesian.ts` there was an error of mine, which I have removed.
So the structure of the options is:
```
  options = {
    width,
    height,
    axes,
    timeScale: {...},
    ...
  }
```
Since the `timeScale` option has always been out of `axes`, there is no need for any warning because the structure has not changed and consequently nothing breaks :)

Reply:
Great thanks Ilaria 🙂